warehouse/query-core/src/main/java/datawave/query/jexl/visitors/HtmlDecorator.java

```diff
@@ -0,0 +1,197 @@
+package datawave.query.jexl.visitors;
+
+import java.util.Collection;
+
+import org.apache.commons.jexl2.parser.ASTAdditiveOperator;
+import org.apache.commons.jexl2.parser.ASTAndNode;
+import org.apache.commons.jexl2.parser.ASTAssignment;
+import org.apache.commons.jexl2.parser.ASTDivNode;
+import org.apache.commons.jexl2.parser.ASTEQNode;
+import org.apache.commons.jexl2.parser.ASTERNode;
+import org.apache.commons.jexl2.parser.ASTFalseNode;
+import org.apache.commons.jexl2.parser.ASTFunctionNode;
+import org.apache.commons.jexl2.parser.ASTGENode;
+import org.apache.commons.jexl2.parser.ASTGTNode;
+import org.apache.commons.jexl2.parser.ASTIdentifier;
+import org.apache.commons.jexl2.parser.ASTLENode;
+import org.apache.commons.jexl2.parser.ASTLTNode;
+import org.apache.commons.jexl2.parser.ASTMethodNode;
+import org.apache.commons.jexl2.parser.ASTModNode;
+import org.apache.commons.jexl2.parser.ASTMulNode;
+import org.apache.commons.jexl2.parser.ASTNENode;
+import org.apache.commons.jexl2.parser.ASTNRNode;
+import org.apache.commons.jexl2.parser.ASTNotNode;
+import org.apache.commons.jexl2.parser.ASTNullLiteral;
+import org.apache.commons.jexl2.parser.ASTNumberLiteral;
+import org.apache.commons.jexl2.parser.ASTOrNode;
+import org.apache.commons.jexl2.parser.ASTSizeMethod;
+import org.apache.commons.jexl2.parser.ASTStringLiteral;
+import org.apache.commons.jexl2.parser.ASTTrueNode;
+import org.apache.commons.jexl2.parser.ASTUnaryMinusNode;
+import org.apache.commons.jexl2.parser.JexlNode;
+
+import datawave.query.jexl.JexlASTHelper;
+import datawave.query.jexl.nodes.QueryPropertyMarker;
+
+/**
+ * Decorates a Jexl Query with HTML coloring. To be used in viewing Jexl Queries on web pages. Specifically, the Query Metrics webpage.
+ */
+public class HtmlDecorator implements JexlQueryDecorator {
+    
+    @Override
+    public void apply(StringBuilder sb, ASTAdditiveOperator node) {
+        sb.append(String.format("<span class=\"add-op\">%s</span>", node.image));
```

Comment:
Probably could have saved yourself a few keystrokes by creating a method to create the CSS tag.

e.g.
```
String createSpan(String cssClass, String content) {
   return String.format("<span class=\"%s\">%s</span>", cssClass, content);
}
```

Then you just call `createSpan(css, content)` everywhere. It also might help if we ever decide to add additional formatting that affects every span.  Adding in one place vs. many.